packages/vuetify/src/components/VTextField/VTextField.sass

```diff
@@ -72,8 +62,22 @@
   fieldset,
   .v-input__control,
   .v-input__slot
+    border-color: inherit
     border-radius: inherit
 
+  fieldset,
+  .v-input__control
+    color: inherit
+
+  &.v-input--has-state
+    .v-input__control > .v-text-field__details > .v-counter
```

Comment:
I was never happy seeing component being styled differently in different context.
So another solution would be to add `hasState` (or `color`) and `disabled` props to v-counter, v-messages, v-label (sounds like a mixin) and simplify styles to
```sass
.v-counter--disabled
  color: $disabled

.v-label--disabled
  color: $disabled
```

Similar `disabled`, color, hover class could be applied to `fieldset`
```sass
.v-text-field__fieldset--disabled
  color: $disabled

.v-text-field__fieldset--hover 
  border: ...

//etc
```

It might become a bigger PR that the current one but imho it would result in a shorter and easier to read and to maintain code

Reply:
Great feedback, I love it.

Reply:
As we discussed on Discord, this may be a feature that we work on later but not a part of this PR. Other than this was there any additional feedback?